branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/manager/IIndentDefining.java

```diff
@@ -0,0 +1,10 @@
+package com.virtuslab.branchlayout.api.manager;
+
+import org.checkerframework.checker.index.qual.Positive;
+
+public interface IIndentDefining {
```

Comment:
`IIndentSpec`?

Reply:
Or maybe `IHasIndentSpec` :cat:

Reply:
Also, I don't think this should/needs to be a part of the API, this sounds like more like a implementation-related piece